yt_dlp/extractor/franceculture.py

```diff
@@ -1,27 +1,39 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
+import re
 from .common import InfoExtractor
 from ..utils import (
     determine_ext,
     extract_attributes,
     int_or_none,
+    traverse_obj,
 )
 
 
 class FranceCultureIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?franceculture\.fr/emissions/(?:[^/]+/)*(?P<id>[^/?#&]+)'
     _TESTS = [{
-        'url': 'http://www.franceculture.fr/emissions/carnet-nomade/rendez-vous-au-pays-des-geeks',
+        # playlist
+        'url': 'https://www.franceculture.fr/emissions/hasta-dente',
+        'playlist_count': 12,
+        'info_dict': {
+            'id': 'hasta-dente',
+            'title': 'Hasta Dente !',
+            'description': None,
```

Comment:
what? If description extraction is failing, it should be fixed, not add `None` to test 😐

Reply:
This never worked before, I'm just trying my best to add a new feature, I'm not the maintainer of that extractor. I will try to fix it, but please be patient with me :)

Reply:
OK fixed it.